examples/koa/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "koa-example",
+  "private": true,
+  "version": "0.8.0",
+  "description": "Example of Koa and @koa/router integration with OpenTelemetry",
+  "main": "index.js",
+  "scripts": {
+    "zipkin:server": "cross-env EXPORTER=zipkin node ./server.js",
+    "zipkin:client": "cross-env EXPORTER=zipkin node ./client.js",
+    "jaeger:server": "cross-env EXPORTER=jaeger node ./server.js",
+    "jaeger:client": "cross-env EXPORTER=jaeger node ./client.js",
+    "lint": "eslint . --ext .js",
+    "lint:fix": "eslint . --ext .js --fix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/open-telemetry/opentelemetry-js-contrib.git"
+  },
+  "keywords": [
+    "opentelemetry",
+    "koa",
+    "tracing",
+    "instrumentation"
+  ],
+  "engines": {
+    "node": ">=8"
+  },
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
+  },
+  "dependencies": {
+    "@koa/router": "^9.3.1",
+    "@opentelemetry/api": "^0.8.3",
```

Comment:
Hi @obecny! I am currently running into some errors with version 0.10.1, but not in my own plugin - 
For the Koa plugin, it is passing all tests using the core version 0.10.2, but was running into some errors with 0.10.1. I think there may be some issues with 0.10.1, at least in my local environment, because I also tried running tests in the Express plugin locally, and they failed as well (see image).

![Screenshot 2020-08-05 at 9 49 01 AM](https://user-images.githubusercontent.com/60523681/89565810-f5323100-d7d3-11ea-9c13-a4c4d95f1599.png)

In the Express plugin, bumping the version to 0.10.2 actually did fix these errors, though I'm not sure it would solve the errors across all plugins currently using 0.10.1. 

Have you been experiencing any issues with the version bump to 0.10.1 in the contrib repo, or would you be willing to help me look into this further? I would really appreciate it

Reply:
@carolinee21 ran down the cause for Express, which might relate to what you've tried/tested.

For express, there was a mismatch between `@opentelemetry/tracing`, `@opentelemetry/core`, and `@opentelemetry/api`.

`@opentelemetry/tracing` was hard-coded to 0.10.1 in the express package.json dev dependencies. The other two are ^0.10.1. So in reality, they were running 0.10.2 while tracing was still on 0.10.1. With 0.10.2, core no-longer exports a `randomSpanId`. Instead, an ID generator is exported to be used (which 0.10.2 of tracing does).

When all of those match, one way or the other, the install/compile/test process passes for me.

Reply:
It looks like this PR *should* solve that for express: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/170/files#diff-d2aa817a652c61faac8e33c6b07d8654

Reply:
@michaelgoin Thank you so much for looking into this! It would be really great if that PR does solve some of these issues, and as soon as it merges I believe this PR should be unblocked!